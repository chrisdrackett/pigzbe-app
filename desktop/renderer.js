import {AppRegistry} from 'react-native';
import App from './bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
