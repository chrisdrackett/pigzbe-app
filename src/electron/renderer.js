import {AppRegistry} from 'react-native';
// import App from '../../App';
import App from '../../dist/bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
