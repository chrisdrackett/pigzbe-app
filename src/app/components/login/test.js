import React from 'react';
import renderer from 'react-test-renderer';
import {LoginComponent} from './';

const props = {
    error: null,
    isLoggingIn: false,
    isLoadingProfile: false
};

describe('Login', () => {
    test('renders correctly', () => {
        const tree = renderer.create(<LoginComponent {...props}/>).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
