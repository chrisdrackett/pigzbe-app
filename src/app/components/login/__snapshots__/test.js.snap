// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "alignSelf": "stretch",
      "backgroundColor": "#003278",
      "flex": 1,
      "justifyContent": "center",
      "paddingTop": 20,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "height": "50vh",
        "justifyContent": "center",
        "width": "100vw",
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../assets/images/header.png",
        }
      }
      style={
        Object {
          "height": "100%",
          "position": "absolute",
          "width": "100%",
          "zIndex": 0,
        }
      }
    />
    <Image
      source={
        Object {
          "testUri": "../../../assets/images/pigzbe_logo.png",
        }
      }
      style={
        Object {
          "height": "63px",
          "width": "162px",
        }
      }
    />
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#ffffff",
          "fontFamily": "Poppins",
          "fontSize": 16,
          "fontWeight": "bold",
          "marginBottom": 20,
          "marginLeft": 20,
          "marginRight": 20,
          "marginTop": 10,
          "position": "relative",
          "textAlign": "center",
          "zIndex": 1,
        }
      }
    >
      Save. Learn. Play.
    </Text>
  </View>
  <View
    style={
      Object {
        "height": "50vh",
        "justifyContent": "center",
        "paddingLeft": "9.375%",
        "paddingRight": "9.375%",
        "paddingTop": "20px",
        "width": "100vw",
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#ffffff",
          "fontFamily": "Poppins",
          "fontSize": 16,
          "fontWeight": "bold",
          "marginBottom": 10,
          "marginLeft": 20,
          "marginRight": 20,
          "textAlign": "center",
        }
      }
    >
      Welcome to Pigzbe
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#ffffff",
          "fontFamily": "Poppins",
          "fontSize": 14,
          "marginBottom": 20,
          "marginLeft": 20,
          "marginRight": 20,
          "opacity": 0.6,
          "textAlign": "center",
        }
      }
    >
      Enter your key to access your Wollo account
    </Text>
    <TextInput
      allowFontScaling={true}
      onChangeText={[Function]}
      placeholder="Your key"
      placeholderTextColor="#97abc7"
      style={
        Object {
          "alignSelf": "stretch",
          "borderColor": "#ffffff",
          "borderRadius": 22.5,
          "borderWidth": 1,
          "color": "#ffffff",
          "fontFamily": "Poppins",
          "fontSize": 14,
          "fontWeight": "bold",
          "height": 45,
          "marginBottom": 10,
          "paddingLeft": 30,
          "paddingRight": 20,
        }
      }
      value="SAXYJU6Q67IXM4DSOFGVJ2L2I7C2SQJSV2MDR2E64AKML5ZXO25RMISJ"
    />
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      collapsable={undefined}
      hasTVPreferredFocus={undefined}
      hitSlop={undefined}
      isTVSelectable={true}
      nativeID={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignSelf": "stretch",
          "backgroundColor": "#FDDF06",
          "borderColor": "#FDDF06",
          "borderRadius": 22.5,
          "borderWidth": 1,
          "marginBottom": 10,
          "opacity": 1,
        }
      }
      testID={undefined}
      tvParallaxProperties={undefined}
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "alignSelf": "stretch",
            "color": "#003278",
            "fontFamily": "Poppins",
            "fontSize": 16,
            "fontWeight": "bold",
            "height": 45,
            "paddingLeft": 20,
            "paddingRight": 20,
            "paddingTop": 10,
            "textAlign": "center",
          }
        }
      >
        Next
      </Text>
    </View>
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      collapsable={undefined}
      hasTVPreferredFocus={undefined}
      hitSlop={undefined}
      isTVSelectable={true}
      nativeID={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
      testID={undefined}
      tvParallaxProperties={undefined}
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "alignSelf": "stretch",
              "color": "#003278",
              "fontFamily": "Poppins",
              "fontSize": 16,
              "fontWeight": "bold",
              "height": 45,
              "paddingLeft": 20,
              "paddingRight": 20,
              "paddingTop": 10,
              "textAlign": "center",
            },
            Object {
              "color": "#b3c1d6",
              "fontSize": 14,
              "textDecorationColor": "#ffffff",
              "textDecorationLine": "underline",
              "textDecorationStyle": "solid",
            },
          ]
        }
      >
        Help?
      </Text>
    </View>
  </View>
</View>
`;
